src/components/ChatInput.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+function ChatInput({ currentUser, setChatList }) {
+  const [message, setMessage] = useState('');
+  const handleInputChange = (e) => {
+    setMessage(e.target.value);
+  };
+  const handleSendMessageBtnClick = () => {
+    if (message)
+      setChatList((previousChat) => [
+        ...previousChat,
+        {
+          text: message,
+          userId: currentUser,
+        },
+      ]);
```

Comment:
```suggestion
      setChatList([
        ...chatList,
        {
          text: message,
          userId: currentUser,
        },
      ]);
```

이렇게 쓰시는게 더 깔끔할 것 같아요. 불필요하게 arrow function이 들어간 감이 있네요.